va/drm/va_drm_utils.c

```diff
@@ -41,8 +41,8 @@ static const struct driver_name_map g_driver_name_map[] = {
     { "pvrsrvkm",   8, "pvr"    }, // Intel UMG PVR driver
     { "emgd",       4, "emgd"   }, // Intel ECG PVR driver
     { "hybrid",     6, "hybrid" }, // Intel OTC Hybrid driver
-    { "nouveau",    7, "gallium" }, // Mesa Gallium driver
-    { "radeon",     6, "gallium" }, // Mesa Gallium driver
+    { "nouveau",    7, "nouveau"  }, // Mesa Gallium driver
+    { "radeon",     6, "r600"     }, // Mesa Gallium driver
```

Comment:
Looking a bit more closely, there doesn't appear to be any released version of Mesa which used the gallium_drv_video.so name.  The change above happened after the 12 branchpoint, but before the actual 12 release.  Versions before 12 did not include the gallium VAAPI driver at all.